pkgs/data/sgml+xml/schemas/sgml-dtd/docbook/4.2.nix

```
@@ -0,0 +1,42 @@
+{ lib, stdenv, fetchurl, unzip }:
+
+let
+
+  src = fetchurl {
+    url = "http://www.oasis-open.org/docbook/sgml/4.2/docbook-4.2.zip";
+    sha256 = "Z+vSyUs0JxjGhl0t5g9dT/Atd6fksNnnKkjEXysmNcM=";
+  };
+
+  isoents = fetchurl {
+    url = "http://www.oasis-open.org/cover/ISOEnts.zip";
```

```suggestion
    url = "https://web.archive.org/web/20220713122518/http://xml.coverpages.org/ISOEnts.zip";
```
http://www.oasis-open.org/cover/ISOEnts.zip redirects to https://www.oasis-open.org/cover/ISOEnts.zip which has a broken cert and then redirects to http://xml.coverpages.org/ISOEnts.zip

---

Cert serms okay now.